src/exports.jl

```diff
@@ -239,6 +240,8 @@ export
   TagType,
   @TagType_str,
   op,
+  op!,
```

Comment:
Do we really need to export this? Since it is mostly if not entirely for overloading, someone would need to import it or define their overload with:
```
function ITensors.op!(...)
  [...]
end
```
anyway. Just trying to be conservative about exports for these earlier versions of the code.